gym/utils/step_api_compatibility.py

```diff
@@ -0,0 +1,180 @@
+"""Contains methods for step compatibility, from old-to-new and new-to-old API, to be removed in 1.0."""
```

Comment:
I was considering this a wrapper, one for `NewStepAPI` and `OldStepAPI` instead of these two functions and modifying the `Wrapper` class

Reply:
I don't get you :thinking: